src/legacy/routes/Host/HostBookingCard/HostBookingCard.tsx

```diff
@@ -14,6 +11,9 @@ import {
 } from 'utils/bookingsDisplayStatus';
 import { currencyToDisplay } from 'utils/currencyToDisplay';
 import { formatDateRange } from 'utils/formatDate';
+import { Card, Col, Row } from 'reactstrap';
+import Loading from '../../../shared/loading/Loading';
```

Comment:
can we use the `/legacy/shared/etc` import here? will be easer to change all occurrences later

Reply:
👍